src/main.c

```diff
@@ -15,52 +15,66 @@
 #include "stm32f0xx_exti.h"
 #include "stm32f0xx_syscfg.h"
 #include "stm32f0xx.h"
+#include "stm32f0xx_adc.h"
 
 #include "FreeRTOSConfig.h"
 
 #include "FreeRTOS.h"
 #include "task.h"
 
+#include "ADC.h"
+
 // Sample pragmas to cope with warnings. Please note the related line at
-// the end of this function, used to pop the compilfer diagnostics status.
+// the end of this function, used to pop the compiler diagnostics status.
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wunused-parameter"
 #pragma GCC diagnostic ignored "-Wmissing-declarations"
 #pragma GCC diagnostic ignored "-Wreturn-type"
 
 
-
 void blinkyTask(void *dummy){
 	RCC_AHBPeriphClockCmd(RCC_AHBPeriph_GPIOC, ENABLE);
 	GPIOC->MODER |= 0x50000;
-	GPIOC->ODR |= 0x100; // 0001 0000 0000
+	GPIOC->ODR |= 0x100;
 	vTaskDelay(2000);
-	GPIOC->ODR &= ~(0x100); //clears pin 8 (0b0000 0000 0000)
+	GPIOC->ODR &= ~(0x100);
 	while(1){
 		GPIOC->ODR ^= 0x200;
-
-		/*0 1 2 3 4 5 6 7 8 9 A  B  C  D  E  F
-		0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
-		0b1111 -> 15 -> 0xF
-
-		0x3B5FA0 -> (# of bytes) = (number of digits) / 2*/
-
 		vTaskDelay(500);
 	}
 }
 
+void ADCTask(){
+	uint16_t i = 0;
+	while(1){
+		uint32_t value = Get_ADC_Channel(i);
+		i++;
+		i %= 8;
+	}
+}
+
```

Comment:
Would you be able to remove this task for the final revision? Just cuz it's a thing used for testing, and we don't really need it in the final revision.